plugins/GSdx/GSState.cpp

```diff
@@ -465,6 +465,21 @@ GSVector4i GSState::GetFrameRect(int i)
 	return rectangle;
 }
 
+int GSState::GetFramebufferHeight()
+{
+	const GSVector4i output[2] = { GetFrameRect(0), GetFrameRect(1) };
+	// Framebuffer height is 11 bits max according to GS user manual
+	const int height_limit = (1 << 11) - 1;
+	int top = std::min(output[0].top, output[1].top);
+	int bottom = std::max(output[0].bottom, output[1].bottom);
+	// According to Gregory, GS seems to wrap when values are passed over limit.
+	int frame_memory_height = (bottom - top) % height_limit;
```

Comment:
Is this supposed to have a range of 0-2047? You have `const int height_limit = (1 << 11) - 1;` or 2047, so the current range is 0-2046.